ansible/www-standalone/resources/scripts/cdn-purge.sh.j2

```diff
@@ -27,6 +27,8 @@ if ! [ -f /tmp/cdnpurge.$site ]; then
   exit 0
 fi
 
+reason="$(cat /tmp/cdnpurge.$site | tr '\n' ' ')"
```

Comment:
Previously this file was a zero-length signal file, now it should contain a "reason", but it may also contain multiple reasons if more than one request was made before we check it, hence the flattening of multiple lines into one. We use this in the log, below.